proteus/TwoPhaseFlow/utils/Parameters.py

```diff
@@ -489,7 +433,7 @@ def _initializeNumerics(self):
         # TIME
         if self.timeDiscretization=='vbdf':
             self.n.timeIntegration = TimeIntegration.VBDF
-            self.n.timeOrder = self.p.CoefficientsOptions.timeOrder
+            self.n.timeOrder = 2
```

Comment:
Is this always true? I was under the impression that `vbdf` was a scheme such that different orders can be used?

Reply:
Not necessarily, let me correct that